tests/runner/origin/base_host.py

```
@@ -329,176 +342,159 @@ class RunHostAndProgramRes:
-	cwd: Path | None = None,
-	exit_timeout=0.05,
-	deadline: float | None = None,
+	timeout: float | None = None,
+	manager_uri: str = 'http://127.0.0.1:3999',
+	logger: Logger | None = None,
+	is_sync: bool,
+	capture_output: bool = True,
+	message: typing.Any,
+	host_data: str = '',
+	host: str,
+	extra_args: list[str] = [],
```

Ruff (0.13.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)